actioncable/lib/action_cable/connection/base.rb

```diff
@@ -172,6 +177,8 @@ def handle_open
           server.add_connection(self)
         rescue ActionCable::Connection::Authorization::UnauthorizedError
           respond_to_invalid_request
+        rescue => e
```

Comment:
This `rescue` could be moved to the app `ActionCable::Connection::Base#connect` so every app handles the exception as it considers appropriate. Still, I think we should be able to use a different close code which, currently, is not possible.